src/app/content/components/SectionHighlights.tsx

```diff
@@ -86,7 +85,8 @@ export const HighlightSection = styled.div`
   }
 
   @media print {
-    page-break-after: avoid;
+    break-after: avoid;
+    break-inside: avoid;
```

Comment:
I've changed it because i've noticed sometimes printing wanted to break `HighlightSection` between pages which looked weird. It wasn't that noticable in MH because their backgrounds are disabled for printing.